CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Versions
 * AWS : adding terraform, tfenv and associated tools. Removing boto3. Bumping misc tools' verions
 * Azure : adding terraform, tfenv and associated tools. Bumping misc tools' verions
 * Terraform : dropping this image. Terraform tooling is now included directly in AWS & Azure images
+* Golang : Bump Golang-ci-lint version
```

Comment:
could you please insert the line between Azure & Terraform, so that the changelog stays sorted ?

Reply:
Woops, sorry, my merge and your review crossed